src/openlifu/db/database.py

```diff
@@ -193,6 +193,10 @@ def write_subject(self, subject, on_conflict=OnConflictOpts.ERROR):
         self.logger.info(f"Added subject with ID {subject_id} to the database.")
 
     def write_transducer(self, transducer, registration_surface_model_filepath: Optional[str] = None, transducer_body_model_filepath: Optional[str] = None, on_conflict: OnConflictOpts=OnConflictOpts.ERROR):
+        """ Writes a transducer object to database and copies the affiliated transducer data files i.e. transdcer body and
+        registration surface model files to the database.These model files are optional data attributes that are only required if running
+        transducer tracking. When a transducer that is already present in the database is being re-written,
+        the associated transducer model files do not need to be provided """
```

Comment:
It would be good to explain in this docstring what is meant by registration surface and body model, because it's really not clear just from the variable name what's the intention.

I suggest a docstring style where you lay out all the args and return value one by one like we have been doign with some of the other functions